gpt4all-chat/llm.cpp

```diff
@@ -67,6 +67,8 @@ bool LLM::checkForUpdates() const
     QString tool("maintenancetool.exe");
 #elif defined(Q_OS_DARWIN)
     QString tool("../../../maintenancetool.app/Contents/MacOS/maintenancetool");
+#elif defined(Q_OS_FREEBSD)
```

Comment:
Yeah, this is not necessary. You don't have an actual installer built from Qt installer framework with FREEBSD so this does nothing.